web/client/configs/localConfig.json

```diff
@@ -426,7 +426,13 @@
                         "alwaysVisible": true
                     }
                 }
-            }, "Home", "FeatureEditor",
+            }, "Home",
+            {
+              "name": "FeatureEditor",
+              "cfg": {
+                "showPopoverSync": false
```

Comment:
this is superfluous

generally speaking the less we edit the cfg the better, because it might requires an update for all our project. instead if we change behaviour like this, we can simply change the default value used in the plugin props
so please remove this change here